composer.json

```diff
@@ -2,7 +2,9 @@
     "name": "postcode-nl/api-magento2-module",
     "description": "Postcode.nl International Address API module for Magento 2. Adds autocompletion for addresses in multiple countries using official postal data.",
     "require": {
-      "php": "7.*|~7.0.6"
+      "php": "7.*|~7.0.6",
+      "magento/module-checkout": "100.2.*",
+      "magento/module-ui": "101.0.*"
```

Comment:
Magento 2.4.2 is 101.2.2, so this means this module cannot be installed with Magento 2.4 right now.

Reply:
Thanks Barry, the dependency will be fixed in the next release.